.github/workflows/publish.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
```

Comment:
:0 wow this is nice

Reply:
assuming it works, havent tried this one yet